README.md

```diff
@@ -46,3 +47,13 @@ $ pipenv run dev
 4. Write Test (so I can save time running the development server and sending stupid requests)
 5. Document API with text-based REST-Client
 ```
+
+## Running
+1. Build docker-compose
+```
+sudo docker-compose up
```

Comment:
`docker-compose up -d` for detachment nhé

Reply:
flag này có tác dụng gì ạ

Reply:
à em đọc được rồi, chạy trên background hả anh

Reply:
tại thấy mấy lần chạy nó cứ chiếm terminal phải mở terminal khác @@